client/src/components/logo/logo.tsx

```diff
@@ -6,16 +6,19 @@ interface Props {
   variant?: 'primary' | 'secondary'
 }
 
-const Logo: FC<Props> = ({ onClick, variant }) => {
+const Logo: FC<Props> = ({ variant }) => {
   return (
-    <Box onClick={onClick}>
+    <Box>
+      <a href="https://www.bigohhh.com/" style={{ textDecoration: 'none' }}>
       <Typography
         variant="h4"
         component="h1"
         sx={{ fontWeight: 700, '& span': { color: variant === 'primary' ? 'primary.main' : 'unset' } }}
-      >
+      > 
         Big<span>Ohhh</span>
-      </Typography>
+      </Typography>        
+      </a>
+
```

Comment:
You can remove this space

Reply:
Okay, @omrajsharma. Could you also tell me how to fix this? I added an <a> tag (i.e., anchor element) inside the <box> element. I set the path of the <a> tag to https://www.bigohhh.com/. Could you please guide me? This is my first time contributing. I am newbies.


I removed the space. Please check it. If it’s okay, then I will have to send you another pull request.
import React, { FC } from 'react'
import { Box, Typography } from '@mui/material'

interface Props {
  onClick?: () => void
  variant?: 'primary' | 'secondary'
}

const Logo: FC<Props> = ({ variant }) => {
  return (
    <Box>
      <a href="https://www.bigohhh.com/" style={{ textDecoration: 'none' }}>
      <Typography
        variant="h4"
        component="h1"
        sx={{ fontWeight: 700, '& span': { color: variant === 'primary' ? 'primary.main' : 'unset' } }}
      > 
        Big<span>Ohhh</span>
      </Typography>        
      </a>
    </Box>
  )
}

Logo.defaultProps = {
  variant: 'primary',
}

export default Logo